ui/dashboards/src/components/Panel/Panel.test.tsx

```diff
@@ -42,19 +43,21 @@ describe('Panel', () => {
     definition ??= createTestPanel();
 
     renderWithContext(
-      <TemplateVariableProvider
-        initialVariableDefinitions={[
-          {
-            kind: 'TextVariable',
-            spec: {
-              name: 'foo',
-              value: 'bar ',
+      <TimeRangeProvider initialTimeRange={{ pastDuration: '1h' }}>
```

Comment:
It means the TemplateVariableProvider cannot be used without being under a time range provider? What is the error message in this case ?

Reply:
Yep, you need the Time Range provider because of the `$__from`/`$__to` variables. It will print something like this: No provider found... Did you forget to import a Provider...